.travis.yml

```diff
@@ -11,6 +11,54 @@ env:
     - CONDA_PY=3.7
   global:
     - PYSAM_LINKING_TEST=1
+    - TWINE_USERNAME=
+    - secure: ''
+
+_deploy_common: &deploy_common
+  if: tag IS present
```

Comment:
I believe you should be able to deploy from master with
`if: branch = master AND type = push AND fork = false` based on https://github.com/travis-ci/travis-ci/issues/8767#issuecomment-344751779. You'd have to revert that after the successful deployment to limit to deployments on tag

Reply:
Thanks, @mvdbeek , I will give this a go.

Reply:
It worked, wheels have been uploaded to PyPI! You can now revert commit 0401ebed8cdce01b0f7da0b47fd41718b4a7e2ec .